spec/factories.rb

```diff
@@ -645,9 +645,6 @@
 
 FactoryBot.modify do
   factory :stock_location, class: Spree::StockLocation do
-    # keeps the test stock_location unique
-    initialize_with { Spree::StockLocation.find_or_create_by_name(name)}
-
```

Comment:
are you sure this is not causing build errors?
I see "Validation failed: The default location already exists"

Reply:
Then we need to fix those because the violate a constraint.